tests/rptest/services/franz_go_verifiable_services.py

```diff
@@ -93,6 +97,26 @@ def stop_node(self, node):
         if self._exception is not None:
             raise self._exception
 
+    def clean_node(self, node):
+        self._redpanda.logger.info(f"{self.__class__.__name__}.clean_node")
+        node.account.kill_process("si-verifier", clean_shutdown=False)
```

Comment:
If we run produce and consumer together, looks like clean method kill both of them or kill randome service?

Reply:
clean_node has to be a bit indiscriminate, because it is used at the start of tests to clear out any mess from previous tests (we don't know the PID in that case).  To make that work when we're sharing the same node for several services, it means that during the test I need to be careful to pass clean=False to the start() calls (apart from the first one in the test).